app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,57 @@
+class Api::V1::CommentsController < ApplicationController
+  before_action :set_post, only: %i[show update destroy]
+
+  # GET /users/:user_id/posts/:post_id/comments/id
```

Comment:
- [ ] This comment is a bit misleading because this index action for `GET` all comments does not take comment  `id` at the end since it returns all comments for a particular post and a particular user. it should be like so 👇🏾 

# GET /users/:user_id/posts/:post_id/comments